commons/src/main/java/com/vitorpamplona/amethyst/commons/richtext/RichTextParser.kt

```diff
@@ -112,20 +127,23 @@ class RichTextParser {
     ): RichTextViewerState {
         val urlSet = parseValidUrls(content)
 
+        val base64Images = parseBase64Images(content)
```

Comment:
This might be really slow. I would run it on the wordIdentifier. Check if each word starts with `data:image` and if so go for the full parse. It avoids running the entire regex when no base64 images can be found.